src/components/MissionContainer.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchdata } from '../redux/Mission/Mission';
+import MissionItem from './MissionItem';
+
+const Missions = () => {
+  const dispatch = useDispatch();
+  const { data, status } = useSelector((state) => state.mission);
+
+  useEffect(() => {
+    if (status === 'idle') {
+      dispatch(fetchdata());
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
- [Optional] Please avoid disabling Linters in your project. Always fix the bugs they flag. There are great tools.